### PR TITLE
feat: Downloads page UI update

### PR DESCRIPTION
**Pull request by tewshi**

### CODE REVIEW

File: pages/download.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+useHead({
+  title: 'download',
+  meta: [
+    {
+      key: 'description',
+      name: 'description',
+      content: 'Download rotki',
+    },
+    noIndex(),
```
Comment (kelsos) [inline pages/download.vue:15]: This we should probably remove since we want the page to be indexed.

File: components/footer/FooterLegalese.vue [hunk 2]
```diff
@@ -4,13 +4,15 @@ const { t } = useI18n();
...
       {{ t('footer_legalese.impressum') }}
     </ButtonLink>
-    <ButtonLink size="lg" to="/privacy-policy">
+    <ButtonLink size="lg" to="/privacy-policy" highlight-active>
       {{ t('footer_legalese.privacy_policy') }}
     </ButtonLink>
-    <ButtonLink size="lg" to="/tos">{{ t('footer_legalese.tos') }}</ButtonLink>
+    <ButtonLink size="lg" to="/tos" highlight-active>{{
+      t('footer_legalese.tos')
+    }}</ButtonLink>
```
Comment (kelsos) [inline components/footer/FooterLegalese.vue:15]:
```suggestion
    <ButtonLink size="lg" to="/tos" highlight-active>
      {{ t('footer_legalese.tos') }}
    </ButtonLink>
```

small nitpick

**Review by kelsos — COMMENTED** [review]
It looks great @tewshi I left two minor comments, now it remains to get a few screenshots and merge it

File: pages/download.vue (hunk 1 above)
Comment (tewshi) [reply pages/download.vue:15]: got it, will do.

**Pull request merged**